MovingBlocks/Terasology
# The game details screen now shows the world name instead of "main"

### Contains

Fixes #3583 by showing the world name ("Perlin-0"/"Flat-1"/etc.) instead of "main".

![gamedetailsscreen](https://user-images.githubusercontent.com/30442236/50434909-08f58180-0894-11e9-9440-edbfabef1744.png)

### How to test

- Create a multi-world game (via "Advanced" set-up) and add multiple worlds.
- Start playing the game and exit to main menu for the game to be saved.
- Select your saved game and go to "Details".
- See the "Worlds & Biomes" tab and note the names on the list under the "Worlds" heading.

### Outstanding before merging

- [ ] Still could allow for custom names (where the player can name the world what they would like instead of "Perlin-1").
